operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/ibm-spectrum-scale-csi-operator.package.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 channels:
-- currentCSV: ibm-spectrum-scale-csi-operator.v2.0.0
+- currentCSV: ibm-spectrum-scale-csi-operator.v2.1.0
```

Comment:
This is wrong, the directory needs to be updated to v2.1.0 first, this will automatically  fail courier tests.